pom.xml

```diff
@@ -2088,6 +2088,112 @@
                 <skipBenchmarks>true</skipBenchmarks>
             </properties>
         </profile>
+        <profile>
+            <id>dev</id>
+            <modules>
+                <module>compiler/ballerina-lang</module>
+
+                <module>stdlib/builtin</module>
+                <module>stdlib/utils</module>
+                <module>stdlib/runtime-api</module>
+                <module>stdlib/test-common</module>
+                <module>stdlib/auth</module>
+                <module>stdlib/cache</module>
+                <module>stdlib/config-api</module>
+                <module>stdlib/crypto</module>
+                <module>stdlib/encoding</module>
+                <module>stdlib/file</module>
+                <module>stdlib/openapi</module>
+                <module>stdlib/io</module>
+                <module>stdlib/socket</module>
+                <module>stdlib/internal</module>
+                <module>stdlib/log-api</module>
+                <module>stdlib/transactions</module>
+                <module>stdlib/llvm</module>
+                <module>stdlib/jvm</module>
+                <module>stdlib/bir</module>
+                <module>stdlib/math</module>
+                <module>stdlib/mime</module>
+                <module>stdlib/reflect</module>
+                <module>stdlib/system</module>
+                <module>stdlib/task</module>
+                <module>stdlib/time</module>
+                <module>stdlib/http</module>
+                <module>stdlib/grpc</module>
+                <module>stdlib/filepath</module>
+                <module>stdlib/websub</module>
+                <module>stdlib/jms</module>
+                <module>stdlib/observability</module>
+                <module>stdlib/database/h2</module>
+                <module>stdlib/database/mysql</module>
+                <module>stdlib/database/sql</module>
+                <module>stdlib/streams</module>
+                <module>stdlib/privacy</module>
+                <module>stdlib/messaging/activemq-artemis</module>
+                <module>stdlib/messaging/rabbitmq</module>
+                <module>stdlib/nats</module>
+
+                <module>misc/lib-creator</module>
+
+                <module>bvm/ballerina-config</module>
+                <module>bvm/ballerina-core</module>
+                <module>bvm/ballerina-runtime</module>
+                <module>bvm/ballerina-streaming</module>
+                <module>bvm/ballerina-logging</module>
+                <module>cli/ballerina-launcher</module>
+                <module>cli/ballerina-packerina</module>
+                <!--<module>compiler/ballerina-backend-llvm</module>-->
```

Comment:
Better to remove all commented modules since these will not be used in the future as well.